odoo/http.py

```diff
@@ -507,8 +507,15 @@ def decorator(f):
             else:
                 routes = [route]
             routing['routes'] = routes
+
         @functools.wraps(f)
         def response_wrap(*args, **kw):
+            # if controller cannot be called with extra args (utm, debug, ...), call endpoint ignoring them
+            spec = inspect.getargspec(f)
+            if not spec.keywords:
+                ignored = ['<%s=%s>' % (k, kw.pop(k)) for k in kw.copy() if k not in spec.args]
```

Comment:
why copying the whole dict?
```suggestion
                ignored = ['<%s=%s>' % (k, kw.pop(k)) for k in list(kw) if k not in spec.args]
```